cryptus-app/components/wallet_viewer/profile_wallet_viewer/collection_block/collection_nfts/collection_nfts.module.scss

```diff
@@ -17,32 +15,23 @@
   border-width: 2px;
   border-radius: 10px;
   border-color: #333333;
-  // padding: min(10%, 128px);
   white-space: nowrap;
   width: 150px;
   height: 150px;
 }
 
 .grid2x2 {
-  height: 100%;
-  width: 100%;
-  display: flex;
-  flex-wrap: wrap;
-  flex-direction: row;
-  justify-content: space-evenly;
+  width:100%;
+  height:100%;
+  display:grid;
+  grid-template-columns: 50% 50%;
+  grid-column-gap: 0;
+  grid-row-gap: 0;
```

Comment:
use grid-template-areas in every grid container, and grid-area in every grid item.

This way, it will be easier to see what is happening and place the items in the correct place.